.travis.yml

```diff
@@ -27,6 +27,7 @@ matrix:
           packages:
           - gnutls-dev
 #    - env: variant=debug tests=1 crypto=wolfssl toolset=gcc sanitizer=on
+    - env: variant=debug tests=1 crypto=openssl webtorrent=on toolset=gcc sanitizer=on
```

Comment:
instead of adding a new job, I think it's fine to enable webtorrent on an existing one. ideally on both a linux and mac os job

Reply:
OK I'll enable it on two existing builds!